examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,520 @@
+r"""
+.. _ref_vm6_example:
+
+Pinched Cylinder
+----------------
+Problem Description:
+ - A thin-walled cylinder is pinched by a force :math:`F` at the middle
+   of the cylinder length. Determine the radial displacement :math:`\delta`
+   at the point where the force :math:`F` is applied.
+   The ends of the cylinder are free edges.
+
+Reference:
+ - R. D. Cook, Concepts and Applications of Finite Element Analysis, 2nd Edition,
+   John Wiley and Sons, Inc., New York, NY, 1981, pp. 284-287.
+   H. Takemoto, R. D. Cook, "Some Modifications of an Isoparametric Shell
+   Element", International Journal for Numerical Methods in Engineering, Vol.7
+   No. 3, 1973.
+
+Analysis Type(s):
+ - Static Analysis ``ANTYPE=0``
+
+Element Type(s):
+ - 4-Node Finite Strain Shell Elements (SHELL181)
+ - 8-Node Finite Strain Shell Elements (SHELL281)
+
+.. image:: ../../_static/vm6_setup.png
+   :width: 400
+   :alt: VM6 Pinched Cylinder Problem Sketch
+
+Material Properties
+ - :math:`E = 10.5 \cdot 10^6 psi`
+ :math:`Nu = 0.3125`
+
+Geometric Properties:
+ - :math:`l = 10.35  in`
+ - :math:`r = 4.953  in`
+ - :math:`t = 0.094  in`
+
+Loading:
+ - :math:`F = 100  lb`
+
+Analysis Assumptions and Modeling Notes:
+ - A one-eighth symmetry model is used. One-fourth of the load is applied
+   due to symmetry.
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm6_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start mapdl and clear it.
+mapdl = launch_mapdl()
+mapdl.clear()
+
+# Enter verification example mode and the pre-processing routine.
+mapdl.verify()
+mapdl.prep7()
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a shell-type).
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Define element number.
+elem_num = 1
+
+# Element type: SHELL181.
+mapdl.et(elem_num, "SHELL181")
+
+# Special Features are defined by keyoptions of shell element:
+
+# KEYOPT(3)
+# Integration option:
+# Full integration with incompatible modes.
+mapdl.keyopt(elem_num, 3, 2)  # Cubic shape function
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material.
+
+# Define material number.
+mat_num = 1
+
+# Define material properties.
+mapdl.mp("EX", mat_num, 10.5E6)
+mapdl.mp("NUXY", mat_num, 0.3125)
```

Comment:
Let's describe which material and units we're using here.

Reply:
Okay, I will add it.
But just in case, I have described all inputs above.